lib/rust/web/src/binding/mock.rs

```diff
@@ -379,6 +379,10 @@ impl<T: ?Sized> AsRef<JsValue> for Closure<T> {
     }
 }
 
+impl<T: ?Sized> Drop for Closure<T> {
+    fn drop(&mut self) {}
+}
+
```

Comment:
Why is this needed?

Reply:
No longer needed. This was used to avoid a Clippy lint, but should be solvable another way.